_j/oom.md

```diff
@@ -8,14 +8,19 @@ allocated resources were exceeded during the resolution process.
 
 ## Issue description
 
-The error can occur in two main cases:
+The error can occur in three main cases:
 
  * The software stack is too large to be resolved given the server side adviser
    configuration.
 
  * There are no valid resolutions for the given software stack and adviser
    failed to explore all the states generated to justify this fact.
 
+ * Not all the transitive dependencies were solved yet and resolver fails to
+   find a valid resolution path. This happens for larger software stacks for
```

Comment:
`large`?

Reply:
Fixed in Removing package ('setuptools', '18.0', 'https://pypi.org/simple') due to build time error in the software environment